pkgs/applications/networking/instant-messengers/jami/default.nix

```diff
@@ -78,11 +100,107 @@ rec {
     enablePushNotifications = true;
   };
 
-  jami-daemon = callPackage ./daemon.nix {
-    inherit version src udev jack jami-meta pjsip-jami opendht-jami;
+  daemon = stdenv.mkDerivation {
+    pname = "jami-daemon";
+    inherit src version meta;
+    sourceRoot = "source/daemon";
+
+    nativeBuildInputs = [
+      autoreconfHook
+      pkg-config
+      perl
+    ];
+
+    buildInputs = [
+      alsa-lib
+      asio
+      dbus
+      dbus_cplusplus
+      fmt
+      ffmpeg_5
+      gmp
+      gnutls
+      http-parser
+      jack
+      jsoncpp
+      libarchive
+      libgit2
+      libnatpmp
+      libpulseaudio
+      libupnp
+      yaml-cpp
+      msgpack
+      opendht-jami
+      openssl
+      pjsip-jami
+      restinio
+      secp256k1
+      speex
+      udev
+      webrtc-audio-processing
+      zlib
+    ];
+
+    doCheck = false; # The tests fail to compile due to missing headers.
```

Comment:
Which ones? and it that our problem or an upstream one?

Reply:
Good catch, thanks! It was an upstream issue due to the patched ffmpeg but now we use the original ffmpeg so the tests works now.